### PR TITLE
bugs with chef-profile edit menu/meal screens

### PR DESCRIPTION
**Pull request by Llavrov**

### CODE REVIEW

File: src/components/pages/chefProfile/mealsTab/ChefProfilePageMealsTab.tsx [hunk 1]
```diff
@@ -124,7 +124,14 @@ export default function ChefProfilePageMealsTab({ cookId }: ChefProfilePageMeals
             )}
 
             {selectedTab === 'EDIT' && editMeal.mealId && (
-                <ChefProfilePageEditMeal mealId={editMeal.mealId} onCancel={(): void => setSelectedTab('ALL')} cookId={cookId} />
+                <ChefProfilePageEditMeal
+                    mealId={editMeal.mealId}
+                    onCancel={(): void => {
+                        setSelectedTab('ALL');
+                        void refetch();
```
Comment (CemYil03) [inline src/components/pages/chefProfile/mealsTab/ChefProfilePageMealsTab.tsx:131]: Why would you refetch on cancel? Wouldn't you only refetch if something has changed?
I would suggest an `onClose` and an `onCloseWithChanges` callback. One which onCloseWithChanges would refetch and onClose not.

File: src/components/pages/chefProfile/menusTab/ChefProfilePageMenusTab.tsx [hunk 2]
```diff
@@ -138,7 +149,7 @@ export default function ChefProfilePageMenusTab({ cookId }: ChefProfilePageMenus
                         {visibleMenus.map((menu, index) => (
                             <div
                                 onMouseUp={(event): void => handleClickOnCard(event, menu.menuId)}
-                                className="relative PEMenuCard"
+                                className="relative PEMenuCard editMenu"
```
Comment (CemYil03) [inline src/components/pages/chefProfile/menusTab/ChefProfilePageMenusTab.tsx:152]: What is css classname editMenu supposed to do?

**Review by CemYil03 — CHANGES_REQUESTED** [review]

File: src/components/pages/chefProfile/menusTab/ChefProfilePageMenusTab.tsx (hunk 2 above)
Comment (Llavrov) [reply src/components/pages/chefProfile/menusTab/ChefProfilePageMenusTab.tsx:152]: It's for selecting parent component with edit/delete dropdown on click.

if (!isParentNodeElementHasClass(event, 'editMenu')) setEditMenuOpen(false);

File: src/components/pages/chefProfile/menusTab/createMenuStep2/CreateCookMenuCourse.tsx [hunk 3]
```diff
@@ -55,7 +55,7 @@ export default function CreateCookMenuCourse({ open, meals, onSuccess, onCancel
                             {mealTypes.map((mealType, index) => (
                                 <PETabItem
                                     key={index}
-                                    title={mealType}
+                                    title={`${mealType}FFF`}
```
Comment (CemYil03) [inline src/components/pages/chefProfile/menusTab/createMenuStep2/CreateCookMenuCourse.tsx:58]: What is this about?

File: src/components/pages/chefProfile/menusTab/editMenuStep2/ChefProfilePageEditMenusStep2.tsx [hunk 4]
```diff
@@ -90,6 +91,8 @@ export default function ChefProfilePageEditMenusStep2({ cookId, menu }: ChefProf
         } catch (e) {
             console.error(e);
         }
+
+        onSaveUpdates();
```
Comment (CemYil03) [inline src/components/pages/chefProfile/menusTab/editMenuStep2/ChefProfilePageEditMenusStep2.tsx:95]: Actually we only want to call this if the action was successful. updateGreetingFromKitchen().then(result => result.data.cook.success ...)

File: src/components/pages/chefProfile/personalTab/section5/ChefProfileSection5.tsx [hunk 5]
```diff
@@ -28,7 +28,7 @@ export default function ChefProfileSection5({ chefProfile }: ChefProfileSection5
     const { data, loading } = useQuery(FindLanguagesDocument);
 
     const languages = data?.languages.findAll ?? [];
-    const filteredLanguages = languages.filter((language) => language.title.includes(languageSearchText));
+    const filteredLanguages = languages.filter((language) => language.title.toLowerCase().includes(languageSearchText.toLowerCase()));
```
Comment (CemYil03) [inline src/components/pages/chefProfile/personalTab/section5/ChefProfileSection5.tsx:31]: Perfect, thank you!

**Review by CemYil03 — CHANGES_REQUESTED** [review]

File: src/components/pages/chefProfile/menusTab/createMenuStep2/CreateCookMenuCourse.tsx (hunk 3 above)
Comment (CemYil03) [reply src/components/pages/chefProfile/menusTab/createMenuStep2/CreateCookMenuCourse.tsx]: Still no answer??

**Review by CemYil03 — CHANGES_REQUESTED** [review]

File: src/components/pages/chefProfile/menusTab/editMenuStep2/ChefProfilePageEditMenusStep2.tsx (hunk 4 above)
Comment (CemYil03) [reply src/components/pages/chefProfile/menusTab/editMenuStep2/ChefProfilePageEditMenusStep2.tsx]: Not changed either...

**Review by CemYil03 — APPROVED** [review]

**Pull request merged**